resources/views/front/community-events/_results.blade.php

```diff
@@ -25,6 +25,7 @@
     </thead>
 <tbody>
 @foreach ($model->rankedResults as $entry)
+    @if ($entry->rank !== -1)
```

Comment:
This check should be done in the query `->where('rank', '<>', -1)`, to keep the html clean of business logic